app/models/api/hses.rb

```diff
@@ -16,21 +16,35 @@ def error_type
   def host
     Rails.configuration.x.hses.api_hostname
   end
+
+  def request
+    Rails.logger.debug <<~EODM
+      HSES #{path}:
+      #{response.inspect}
+    EODM
+    if response.failed?
+      Rails.logger.error "HSES call to #{path} responded with #{response.code}"
+    end
+    response
+  end
 end
 
 class Api::Hses::Issue < ApiRequest
   include Api::Hses
-  include FakeApiResponseWrapper
   attr_accessor :id
 
   def initialize(id:)
     @id = id
   end
 
-  # TODO update this with real data once we have /issue endpoint
-  def request
-    fake_issue = FakeIssues.instance.data.find { |c| c[:id] == id }
-    details_response(fake_issue || FakeIssues.instance.data.last)
+  private
+
+  def path
+    "/issues-ws/issue/#{id}"
+  end
+
+  def parameters
+    {types: 1}
```

Comment:
[question] Can you explain why you are sending `types` as part of the URL? I don't see in the spec where we would be sending additional parameters for a specific issue besides the issue id in the path.

Reply:
i dunno, i thought it was needed! but i guess it's not so i will take it out